lib/jsonapi_compliable/base.rb

```diff
@@ -262,20 +262,18 @@ def jsonapi_render_options
       options
     end
 
-    def resolve(base)
+    # Todo: es no need for sep query
```

Comment:
Assume `es` == Elastic Search? If so, what do you mean here, and if not, what?

Reply:
This is just a note to myself - for sql we need a separate query to fetch the stats, not so for elastic or others. We need to provide a way to do both.